packages/titanium-button/src/titanium-button.ts

```diff
@@ -0,0 +1,118 @@
+import '@material/mwc-ripple';
+import { css, customElement, html, LitElement, property } from 'lit-element';
+
+@customElement('titanium-button')
+export class TitaniumButtonElement extends LitElement {
+  @property({ type: Boolean, reflect: true }) disabled: boolean;
+
+  static styles = css`
+    :host {
+      display: inline-flex;
+      position: relative;
+      font-family: Roboto, Noto, sans-serif;
+      -webkit-font-smoothing: antialiased;
+      border-radius: 4px;
+      overflow: hidden;
+      --mdc-theme-primary: var(--titanium-button-text-color, #3b95ff);
+    }
+
+    button {
+      /* Button Reset */
+      border: none;
+      outline: none;
+      margin: 0;
+      width: auto;
+      overflow: visible;
+      background: transparent;
+      /* inherit font & color from ancestor */
+      font: inherit;
+      /* Corrects font smoothing for webkit */
+      -webkit-font-smoothing: inherit;
+      -moz-osx-font-smoothing: inherit;
+      box-sizing: border-box;
+      min-width: 64px;
+      height: 36px;
+      display: block;
+      cursor: pointer;
+      color: var(--titanium-button-text-color, #3b95ff);
+      font-weight: 500;
+      letter-spacing: 1.25px;
+      padding: 8px 16px;
+      border-radius: 4px;
+    }
+
+    focus-veil {
+      position: absolute;
+      display: none;
+      user-select: none;
+      pointer-events: none;
+      top: 0;
+      right: 0;
+      left: 0;
+      background-color: var(--titanium-button-background-color, #3b95ff);
+      bottom: 0;
+      opacity: 0.12;
+    }
+
+    button:active,
+    button:hover {
+      font-weight: 500;
+      outline-style: none;
+      box-shadow: none;
+    }
+
+    button:focus {
+      outline: none;
+    }
+
+    :host([raised]:not(:disabled)) focus-veil {
+      background-color: var(--titanium-button-text-color, #000);
+    }
+
+    button:hover focus-veil {
+      display: block;
+    }
+
+    :host([raised]:not(:disabled)) {
+      padding: 4px;
+    }
+
+    :host([raised]:not(:disabled)) {
+      --mdc-theme-primary: var(--titanium-button-text-color, #000);
+    }
+
+    :host([raised]:not(:disabled)) button {
```

Comment:
disabled raised buttons look the same as enabled raised buttons.
Same with unelevated